packages/nextjs/app/builders/0x2723A2756ecb99b3B50f239782876fB595728AC0/page.tsx

```diff
@@ -0,0 +1,58 @@
+"use client";
+
+import "./page.css";
+import type { NextPage } from "next";
+
+const PersonalPage: NextPage = () => {
+  return (
+    <section className="home section" id="home">
+      <div className="home__container container grid">
+        <div className="home__content grid">
+          <div className="home__social">
+            <a
+              href="https://www.linkedin.com/in/itsarjn"
+              className="home__social-icon"
+              target="_blank"
+              onMouseEnter={() => "ABCD"}
```

Comment:
Lol lets remove this

Reply:
Lol, my bad. I have removed it. I was trying out something. @technophile-04 .